content/1_General/Debugging_Lang.mdx

```diff
@@ -343,45 +419,15 @@ failure?
 		url="https://codeforces.com/blog/entry/15547"
 		starred
 	>
-		includes all the options mentioned below
+		Includes all the options mentioned below.
 	</Resource>
 </Resources>
 
-You can also check what options
-[Errichto](https://codeforces.com/blog/entry/64993) and
-[ecnerwala](https://gist.github.com/ecnerwala/a3c6332ac626bc448165) use.
```

Comment:
^ Leave these in. They're useful (at least to me). Just linking to https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html isn't sufficient because most of the options aren't relevant for competitive programming.

Reply:
alr ig